src/Take.Blip.Builder/Models/Crm.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Take.Blip.Builder.Models
+{
+    /// <summary>
+    /// Crm's lists
+    /// </summary>
+    public enum Crm
+    {
+        /// <summary>
+        /// Sales force crm
+        /// </summary>
+        SalesForce
```

Comment:
An `enum` with a single value isn't really necessary. Will there be more options later?

Reply:
Yes

Reply:
More CRM's will be integrated in the future